spec/services/courses/add_students_service_spec.rb

```diff
@@ -127,5 +127,30 @@
         expect(current_email).to eq(nil)
       end
     end
+
+    context 'notification is enabled' do
+      let(:notify) { true }
+      let(:school) { course.school }
+
+      it 'send email to user set his first password' do
+        service = double('Users::MailFirstPasswordTokenService')
+        allow(Users::MailFirstPasswordTokenService).to receive(:new) { service }
```

Comment:
This is the code we ends up with when there is no proper dependency injection. But let's leave it for now. I want this service to be taken out of this class.